cmaes/_cma.py

```diff
@@ -167,7 +168,7 @@ def __init__(
         self._p_sigma = np.zeros(n_dim)
         self._pc = np.zeros(n_dim)
 
-        self._mean = mean
+        self._mean = copy.deepcopy(mean)
```

Comment:
[nits] I'd say `copy.copy()` or `numpy.copy()` is enough for numpy array. How about using `numpy.copy()` since it's a numpy specialized function? I'm not familiar with the difference though.

```suggestion
        self._mean = mean.copy()
```

Reply:
Thanks! I fixed it now.